x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -327,6 +327,11 @@ export const cli = () => {
           alias: 'c',
           type: 'string',
           default: '',
+        })
+        .option('beforeSpec', {
```

Comment:
nit: Name `beforeSpec` feels a bit vague but I'm not sure what could be the best name here. Leaving a comment would definitely help. It mightn't be really transparent when `beforeSpec` is executed. Different testing frameworks have `onBefore` hook. Maybe the name could `onBeforeHook`. WDYT?

Reply:
Went with `onBeforeHook` :)